e2e/e2eutil/node.go

```diff
@@ -0,0 +1,94 @@
+package e2eutil
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/hashicorp/nomad/helper/uuid"
+)
+
+// NodeRestart is a test helper function that restarts a client node
+// running under systemd using a raw_exec job. Returns the job ID of
+// the restart job so that callers can clean it up.
+func NodeRestart(client *api.Client, nodeID string) (string, error) {
```

Comment:
Can you name this `AgentRestart`? I assumed it was a system reboot from the name.